### PR TITLE
Improved Footer

### PR DESCRIPTION
**Pull request by Swastik1710**
## Issue that this pull request solves:  Improves the footer component.

 Closes: #125

## Types of changes : Representation improvements

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality) Newsletter added
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Final

## Other information

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Swastik1710 Can we please have something like this 
Screenshot from 2020-12-29 13-25-05

Just make the title `Sign Up for our Newsletter` to have same look as `Navigation` text in footer and the background color for bottom-most div is `#016795`

**Comment by Swastik1710** [comment]
Yeah sure. I'll get back to you after making the changes.

**Comment by Swastik1710** [comment]
Hey @Kajol-Kumari , I have made the required changes. Please have a look and let me know if any other changes are required.

File: frontend/src/components/footer/footer.css [hunk 1]
```diff
@@ -150,6 +154,146 @@
...
+
+.cprt_text{
+  background-color: #016795;
+  width: 1349px;
+  color : white;
+  margin-bottom: -19px;
+  margin-top: 30px;
+  margin-left: -50px;
+  padding: 3px 0px; 
+  border-top: 3px solid black;
```
Comment (Kajol-Kumari) [inline frontend/src/components/footer/footer.css:166]: please remove this line and add 
```
  font-size: 14px;
  font-weight: 600;
```

File: frontend/src/components/footer/footer.js [hunk 2]
```diff
@@ -58,12 +64,12 @@ const Footer = () => {
...
           </ul>
         </li>
-        <li class="nav__item">
+        <li>
           <h2 class="nav__title">Social</h2>
           <ul class="nav__ul">
             <li>Be sure to give us a follow on the below social links</li>
           </ul>
-          <a
+          {/* <a
```
Comment (Kajol-Kumari) [inline frontend/src/components/footer/footer.js:72]: please remove these commented lines as we won't require this anymore

File: frontend/src/components/footer/footer.js [hunk 3]
```diff
@@ -76,23 +82,71 @@ const Footer = () => {
...
+                  <i className="fab fa-github  fa-2x"></i>
+                </a>
+              </li>
+            </ul>
+          </div>
         </li>
       </ul>
-      <div className="footer-dash">
-        <div className="footer-text">
+      <div class="footer-dash">
```
Comment (Kajol-Kumari) [inline frontend/src/components/footer/footer.js:136]: replace all `class` with `className`

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Swastik1710 THans for incorporating the requested changes. Please do these minor changes and reduce a bit of space between the contents and that `pink line`. Then it will be good to go :)

**Comment by Swastik1710** [comment]
Hey @Kajol-Kumari , kindly review the changes and let me know if any further changes required.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Swastik1710 Footer is looking great! But somehow the website is getting distorted for small s
Screenshot from 2020-12-30 13-32-12
creens, please fix that and it will be good to go :)

**Comment by Swastik1710** [comment]
@Kajol-Kumari , I've fixed this issue for both mobile and tab view. Please let me know if any further changes are required.

**Comment by Kajol-Kumari** [comment]
Hey @Swastik1710 please resolve the conflict so that I can merge this :)

**Comment by Swastik1710** [comment]
Done.

**Comment by Kajol-Kumari** [comment]
> Done.

Seems like, something got messed up as i am getting blank space at smaller screen

Screenshot from 2020-12-30 18-00-17

**Comment by Swastik1710** [comment]
All done. Please have a look.

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Swastik1710 Why am I seeing 4000+ lines of file change. Please remove the unnecessary chanegs

**Comment by Swastik1710** [comment]
After I resolved the conflicts , when I tried to do git push it showed an error telling me to do Git pull. And after doing git pull when I did git push these two errors showed up:- 
Module not found: Cannot resolve 'styled-component' 
Module not found: Cannot resolve 'semantic-ui-css' 

So I did
npm i semantic-ui-react semantic-ui-css
npm i styled-components

which solved the errors and this only caused 4k+ lines of changes.

So what do you want me to do now ?

**Comment by Kajol-Kumari** [comment]
@Swastik1710 do this 
1. Copy the relevant codes somewhere as I can see you have made the PR using your `main` branch
2. Reset your head with upstream head 
```
git remote add upstream https://github.com/HITK-TECH-Community/Community-Website.git
git fetch upstream
git checkout main
git reset --hard upstream/main  
git push origin main --force
```
This will make your main branch even with the main repository.
After this, make a new branch (you can google it, if you don't know) and then put your code in there, push the change sand then make a new PR.

**Pull request closed without merging**